pyproject.toml

```diff
@@ -34,10 +50,6 @@ repository = "https://github.com/tethysplatform/tethys"
 [project.scripts]
 tethys = "tethys_cli:tethys_command"
 
-[project.optional-dependencies]
-test = [
-    "requests_mock"
-]
-
 [tool.setuptools_scm]
 write_to = "tethys_portal/_version.py"
+local_scheme = "no-local-version"
```

Comment:
what is this?

Reply:
I added this while testing the push of my local build to the Test PyPi repo. The version dynamically created when building the distributions included a local hash of sorts that PyPi doesn't support. Adding that line fixed it. See https://pypi.org/project/setuptools-scm/7.0.3/#version-number-construction.

We can likely remove since it likely won't apply to officially tagged releases, but I figured it also doesn't hurt to keep it in.